intera_interface/src/intera_motion_interface/motion_trajectory.py

```diff
@@ -70,14 +70,22 @@ def __init__(self, label = None,
     def stop_trajectory(self):
         self._client.stop_trajectory()
 
-    def send_trajectory(self):
+    def send_trajectory(self, wait_for_result=True):
```

Comment:
None of the other functions, except for __init__ in this file use the @param description. Maybe we need a general